src/sinks/datadog/metrics/encoder.rs

```diff
@@ -385,7 +385,9 @@ fn sketch_to_proto_message(
     let name = get_namespaced_name(metric, default_namespace);
     let ts = encode_timestamp(metric.timestamp());
     let mut tags = metric.tags().cloned().unwrap_or_default();
-    let host = tags.remove(log_schema.host_key()).unwrap_or_default();
+    let host = tags
+        .remove(log_schema.host_key().unwrap().to_string().as_str())
+        .unwrap_or_default();
```

Comment:
Is the default value here a `None`?

Reply:
The resulting type is `Option<String>`, so yes.